presto-main/src/main/java/com/facebook/presto/sql/planner/plan/TopNRowNumberNode.java

```diff
@@ -57,7 +57,7 @@ public TopNRowNumberNode(
         requireNonNull(source, "source is null");
         requireNonNull(specification, "specification is null");
         requireNonNull(rowNumberSymbol, "rowNumberSymbol is null");
-        checkArgument(maxRowCountPerPartition > 0, "maxRowCountPerPartition must be > 0");
+        //checkArgument(maxRowCountPerPartition > 0, "maxRowCountPerPartition must be > 0");
```

Comment:
Remove code rather than commenting it out.

Reply:
Sorry I didn't know people would review this. It was not a formal fix and I will remove comments in formal PR.